x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/downsample/RollupShardIndexer.java

```diff
@@ -131,6 +135,8 @@ class RollupShardIndexer {
 
     public DownsampleIndexerAction.ShardDownsampleResponse execute() throws IOException {
         long startTime = System.currentTimeMillis();
```

Comment:
Maybe use `client.threadPool().relativeTimeInMillis()` too?
And then when info line is logged use: `client.threadPool().relativeTimeInMillis() - startTime`?